sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnSignature.java

```diff
@@ -234,11 +250,25 @@ public ResultT dispatch(TimerParameter p) {
     }
 
     // These parameter descriptors are constant
+    private static final ContextParameter CONTEXT_PARAMETER =
+        new AutoValue_DoFnSignature_Parameter_ContextParameter();
+    private static final ProcessContextParameter PROCESS_CONTEXT_PARAMETER =
+          new AutoValue_DoFnSignature_Parameter_ProcessContextParameter();
```

Comment:
I think the indentation should agree between these two lines... so this is wrong?